.mergify.yml

```diff
@@ -0,0 +1,8 @@
+pull_request_rules:
+  - name: automatic merge for approved Dependabot pull requests
+    conditions:
+      - author=dependabot-preview[bot]
```

Comment:
we have two conditions for the actor/author on the github action, but only one here... can we check for both here?

```
if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
```

Reply:
we can use a regular expression to match the author name like this.
```yaml
    conditions:
      - author~=\b(\w*dependabot\w*)\b
```
not sure if this is the best regular epression though.

Reply:
Does the regex processor support `|` as "or"? e.g. something like `author~=dependabot\[bot\]|dependabot-preview\[bot\]`

Reply:
The way you have it works. Think it's ready.